modules/virt-creating-and-exposing-mediated-devices.adoc

```diff
@@ -43,81 +48,85 @@ spec:
       - nvidia-233
       nodeSelector:
         kubernetes.io/hostname: node-11.redhat.com
-  permittedHostDevices:
-    mediatedDevices:
-    - mdevNameSelector: GRID T4-2Q
-      resourceName: nvidia.com/GRID_T4-2Q
-    - mdevNameSelector: GRID T4-8Q
-      resourceName: nvidia.com/GRID_T4-8Q
-# ...
+  # ...
 ----
-====
 
-. Create mediated devices by adding them to the `spec.mediatedDevicesConfiguration` stanza:
+ Identify the name selector and resource name values for the devices that you want to expose to the cluster, as shown in the following example. You can use the same value for both, replacing any spaces in the name with an underscore.
 +
-.Example YAML snippet
-[source,yaml]
+[source,terminal]
 ----
-# ...
-spec:
-  mediatedDevicesConfiguration:
-    mediatedDeviceTypes: <1>
-    - <device_type>
-    nodeMediatedDeviceTypes: <2>
-    - mediatedDeviceTypes: <3>
-      - <device_type>
-      nodeSelector: <4>
-        <node_selector_key>: <node_selector_value>
-# ...
+$ oc debug node/node-11.redhat.com
+sh-5.1# chroot /host
+sh-5.1# cd sys/class/mdev_bus
+sh-5.1# ls
+sh-5.1# cd 0000:4b:00.4/mdev_supported_types
+sh-5.1# ls
+sh-5.1# cd nvidia-745
+sh-5.1# ls
+sh-5.1# cat name
```

Comment:
This is technically 9 commands, and we are supposed to include only one command per code block. Additionally, the `ls` commands are not meaningful without example output, because the subsequent commands are based on the `ls` output.